applications/asset_tracker/src/main.c

```diff
@@ -1146,7 +1240,18 @@ void main(void)
 
 	work_init();
 	modem_configure();
+
 connect:
+	k_sem_take(&cloud_connect_ready, K_FOREVER);
```

Comment:
Perhaps it's just me but I got confused about this variable name, it suggests that the connection to the cloud is already done. How about calling it `cloud_ready_to_connect`?

Reply:
Much better, will use that instead, thanks